src/modules/renderGraphiQL.ts

```diff
@@ -95,7 +95,7 @@ export function renderGraphiQL(data: GraphiQLData): string {
       }
     }
     // We don't use safe-serialize for location, becuase it's not client input.
-    var fetchURL = locationQuery(otherParams, ${endpointURL});
+    var fetchURL = locationQuery(otherParams, '${endpointURL}');
```

Comment:
@helfer I still need you to confirm the commit and the next one are needed.  I needed them for the example in the readme to work